composer.json

```diff
@@ -5,11 +5,11 @@
   "license": "MIT",
   "type": "project",
   "require": {
-    "php": ">=7.0",
-    "illuminate/database": "^5.5|^6.0|^7.0|^8.0"
+    "php": ">=7.0|>=8.0.2",
```

Comment:
Is this needed?

Reply:
I don't know, I added it because Laravel 9 requires PHP 8.0.2 explicitly.

Reply:
Ok, can you remove this -- composer should install the `illuminate/database` version that is compatible with the machine's PHP version.